features/hbx_admin/step_definitions/manage_sep_types_steps.rb

```diff
@@ -948,8 +948,7 @@ def fehb_qualifying_life_events
 end
 
 When("Patrick Doe clicked on Shop For Plans button") do
-  sfp = page.all('div').detect { |div| div.text == 'Shop For Plans'}
-  sfp.click
+  find('.pull-right.interaction-click-control-shop-for-plans', text: 'Shop For Plans').click
```

Comment:
If we're going to target text in a cucumber, I think we should use the translation method instead of hardcoding the string: `l10n("insured.shop_for_plans")`